test.js

```diff
@@ -0,0 +1,21 @@
+const validGithubUsername = require("./index");
+const assert = require('assert');
+
+describe("removes illegal characters", function() {
```

Comment:
I've been using single quotes everywhere else, so just use single quotes in the tests, too 👍